docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-classification]]
+=== Predicting delaying flights with {classanalysis}
+
+Let's try to predict whether a flight will be delayed or not by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
+a {classification} model that allows us to make the prediction based on 
+information such as weather conditions and location of the destination and 
+origin, flight distance and carrier.
+
+This dataset consists of sample data that has been manually created and it 
```

Comment:
We can likely shorten this.  Not sure if "made up"/"fabricated"/etc is the right word, but here's a suggestion:
```suggestion
This sample dataset is made up and it 
```